Apps/Contoso.UWP.Demo/App.xaml.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using Microsoft.AppCenter;
+using Microsoft.AppCenter.Analytics;
+using Microsoft.AppCenter.Crashes;
+using Microsoft.AppCenter.Push;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace Contoso.UWP.Demo
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    sealed partial class App : Application
+    {
+        /// <summary>
+        /// Initializes the singleton application object.  This is the first line of authored code
+        /// executed, and as such is the logical equivalent of main() or WinMain().
+        /// </summary>
+        public App()
+        {
+            this.InitializeComponent();
+            this.Suspending += OnSuspending;
+            AppCenter.LogLevel = LogLevel.Verbose;
+            AppCenter.SetLogUrl("https://in-integration.dev.avalanch.es");
+            AppCenter.Start("42f4a839-c54c-44da-8072-a2f2a61751b2", typeof(Analytics), typeof(Crashes), typeof(Push));
```

Comment:
This app has already a store association with the other puppet, and store association matters for Push so we cannot reuse the same app.

I would create another one, also since we use nugets by convention we use appcenter.ms for those apps.